### ISSUE TITLE
Theme breaks youtube with https

### ISSUE THREAD

**Issue by samdobrow** [VisualComposer/visual-composer-starter-child#1]
When the theme is activated embedded YouTube videos load in the iframe but wont stream after the play button is clicked. It works fine under HTTP but breaks on HTTPS.

See this page:  https://samdobrowphotography.com/artist-talk-cuba-living-under-the-shadow/

This was working until recently after some automatic updates. It appears to be a theme conflict.

Additionally, the page works while using the WordPress customizer. This is a site-wide problem with all YouTube videos on the desktop. Mobile works fine.

**Comment by nikitahl** [comment]
Hello!

Could you please provide additional information about your issue?

We couldn't replicate it.

Tested on:
- WordPress: 5.5.1
- OS: macOS Catalina (10.15.5)
- Browsers:
  - Safari 13.1.1
  - Firefox 80.0.1
  - Chrome 85.0.4183.102

Steps were taken:
- Activate **Visual Composer Starter Child** theme
- Create a new post with WordPress
- Add a YouTube block
- Enter YouTube video URL
- Publish post
- View Post
- Click on a YouTube video
- Issue cannot be replicated

**Comment by samdobrow** [comment]
Thank you for your quick response. I have identified the problem and
solution.

The problem was related to browser privacy settings in Firefox and YouTube
tracking. Apparently, YouTube began some type of tracking when embedded
videos are streamed and the browser blocked the stream. Once I turned off
the browser privacy settings the video streamed normally. Next, I found an
obscure setting on the YouTube embed dialog that allowed me to set enhanced
privacy. That setting changed the YouTube URL to include "nocookie" as part
of the domain name. Once the new URL was used in the iframe the video
streamed normally even with the restrictive browser privacy on.

For some reason the tracking cookie did not cause an issue while in
Customizer preview mode. Also the problem did not occur in an old theme. I
am not sure why things changed suddenly. It may have been a core update.
Deactivating all plugins had no effect.

Hope this helps if others experience a similar issue.

Thanks again for your help.
